internal/namespaces/instance/v1/custom_server_test.go

```diff
@@ -279,6 +279,49 @@ func Test_ServerDelete(t *testing.T) {
 	interactive.IsInteractive = false
 }
 
+// These tests needs to be run in sequence
+// since they are using the interactive print
+func Test_ServerTerminate(t *testing.T) {
+	interactive.IsInteractive = true
+
+	t.Run("without IPs", core.Test(&core.TestConfig{
+		Commands:   GetCommands(),
+		BeforeFunc: core.ExecStoreBeforeCmd("Server", "scw instance server create image=ubuntu-bionic -w"),
+		Cmd:        `scw instance server terminate {{ .Server.ID }}`,
+		Check: core.TestCheckCombine(
+			core.TestCheckGolden(),
+			core.TestCheckExitCode(0),
+		),
+		AfterFunc:       core.ExecAfterCmd(`scw instance ip delete {{ index .Server.PublicIP.ID }}`),
+		DisableParallel: true,
+	}))
+
+	t.Run("with IPs", core.Test(&core.TestConfig{
+		Commands:   GetCommands(),
+		BeforeFunc: core.ExecStoreBeforeCmd("Server", "scw instance server create image=ubuntu-bionic -w"),
+		Cmd:        `scw instance server terminate {{ .Server.ID }} with-ip=true`,
+		Check: core.TestCheckCombine(
+			core.TestCheckGolden(),
+			core.TestCheckExitCode(0),
+		),
```

Comment:
We should add a CheckFunc that test that IP GetIP on the API return a not found

Reply:
I just tried, and the problem is: `GetIP` just after a `DeleteIP` returns a 403 instead of 404.

Not sure if this is intentional/stable and should be relied on in tests ?